src/test/scala/chiselTests/Stop.scala

```diff
@@ -4,16 +4,25 @@ package chiselTests
 
 import chisel3._
 import chisel3.testers.BasicTester
+import _root_.circt.stage.ChiselStage
 
 class StopTester() extends BasicTester {
-  stop()
+  chisel3.stop()
+}
+
+class StopWithMessageTester() extends BasicTester {
+  val cycle = RegInit(0.U(4.W))
+  cycle := cycle + 1.U
+  when (cycle === 4.U) {
+    chisel3.stop(p"cycle: $cycle")
```

Comment:
```suggestion
    chisel3.stop(cf"cycle: $cycle")
```
I thought we deprecated `p"..."`, but in any case we should use `cf"..."` in examples